### ISSUE TITLE
Nothing happens

### ISSUE THREAD

**Issue by TheViciousGames** [playgameservices/play-games-plugin-for-unity#2589]
Hello,I published my first game on Play Store.In the final release I tried to implement Google Services Plugin but I'm struggling to do it.I tried many versions,many things but nothing solves it.The problem is that after implementing it in Unity and releasing the app nothings shows,nothing appears,no log in,no nothing.
I tried different plugin versions,different scripts,changed SDK APIs.I did everything right,SHA 1 keys are right.
I'd be happy to get even a failed signing-in.I just want the Google Leaderboard to work.Thanks.

This is my game.
https://play.google.com/store/apps/details?id=com.TheViciousGames.InfiniteJumper

**Comment by dejimarquis** [comment]
I am experiencing the same issue too! I have tried rewriting the code multiple time with different libraries but I get nothing too. This is the only thing I can get to debugging the issue

![image](https://user-images.githubusercontent.com/16407679/59235528-a0cafb80-8ba6-11e9-9058-0eae59f1757c.png)

When I try the app on my phone nothing appears too. No log in, no leaderboard. Please help I have been experiencing this issue for a while now

**Comment by legas1** [comment]
Hi there,

@TheViciousGames what is adb logcat showing?

@dejimarquis it doesn't work in Unity Editor, show logs from adb logcat.

**Comment by TheViciousGames** [comment]
@legas1
I haven't tried logcat yet
I will reply with the answer in a couple of hours.
This is really frustrating.

**Comment by TheViciousGames** [comment]
@dejimarquis any news?

**Comment by TheViciousGames** [comment]
I got news.
I reinstalled the plugin and this time it was different and took longer time to install.
Installed the game from the store on my test phone and it logged.Leaderboards were not working but unfortunately that was the only time I saw something working.
Now i tried another workaround and I'm waiting to test.
Oh and BTW i did a logcat.It says ERROR_NOT_AUTHORIZED.
I'm not using cloud save.

**Comment by avinash2k** [comment]
Have you setup Gameservices properly . The client ID should match when linking the game.

Also make sure you add the Client ID ( The last field that says Optional ) in Window->GooglePlayGames->Setup->Android Setup .

**Comment by Electroblade** [comment]
I have the same problem. When I ran the game on my emulator the google play store sign in box showed up, indicating that the authenticate worked, but nothing else worked. I added buttons to call up the leaderboards and achievements with: Social.ShowAchievementsUI(); and Social.ShowLeaderboardUI(); respectively. They didn't do anything. 

I went through everything and double checked everything and saw that the Java Environmental variables was not setup in windows. I added them and rebooted. Now nothing is showing not even the google authenticate. 

I removed the Plugin and imported it again. I even updated to the new version. I know I must be missing something, but don't know what.
